include/freerdp/utils/gfx.h

```diff
@@ -30,6 +30,8 @@ extern "C"
 {
 #endif
 
+	FREERDP_API const char* rdpgfx_caps_version_str(UINT32 capsVersion);
```

Comment:
Perhaps you could add a javadoc that would contain a "since version 3.X" as it's a public API (I should do the same in my RCG PR BTW)

Reply:
hmm, true, we should add some doxygen stuff.

Reply:
https://github.com/FreeRDP/FreeRDP/issues/10568